statistics/handle/update.go

```diff
@@ -1027,11 +1028,22 @@ func (h *Handle) HandleAutoAnalyze(is infoschema.InfoSchema) (analyzed bool) {
 		return false
 	}
 	pruneMode := h.CurrentPruneMode()
+	rd := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
We may need to change to
```suggestion
	rd := rand.New(rand.NewSource(time.Now().UnixNano())) // #nosec G404
```
to skip the security check.